src/components/Tabs/TabsContext/__tests__/__snapshots__/tabsContext-snapshot-tests.jest.js.snap

```diff
@@ -3,23 +3,26 @@
 exports[`TabsContext renders correctly with activeTabId 1`] = `
 <div
   className="tabs-context--wrapper"
+  data-testid="tabs-context"
 >
   <div
-    className="tabs--wrapper Compact"
+    className="tabsWrapper tabs--wrapper Compact Compact"
+    data-testid="tab-list"
   >
     <ul
-      className="tabs-list"
+      className="tabsList tabs-list undefined"
```

Comment:
have a PR on master to fix such errors https://github.com/mondaycom/monday-ui-react-core/pull/1133